force-app/main/default/aura/HH_Container/HH_ContainerHelper.js

```diff
@@ -440,7 +440,9 @@
             hh.BillingState = addr.MailingState__c;
             hh.BillingPostalCode = addr.MailingPostalCode__c;
             hh.BillingCountry = addr.MailingCountry__c;
-            hh.Undeliverable_Address__c = addr.Undeliverable__c === undefined ? false : addr.Undeliverable__c;
+            if(hh.hasOwnProperty('Undeliverable_Address__c') && addr.hasOwnProperty('Undeliverable__c')){
```

Comment:
This being javascript, I have a feeling that a namespace is needed on the field names in quotes. I remember bumping into this kind of issue in the CRLP Config pages. It's worth testing though, if there's a way to do that.

Reply:
Normally yes, but we use a method `removePrefixFromObjectFields` when loading the household (and the other sobjects) that should allow us to safely ignore the namespace.